pkg/driver/driver.go

```diff
@@ -28,6 +28,7 @@ type Driver interface {
 	Delete() error
 	GetExisting() (string, error)
 	GetVMs(string) (VMs, error)
+	GetVolNames([]corev1.PersistentVolumeSpec) ([]string, error)
```

Comment:
I really wish, if there could be a generic way to fetch this information and we dont require providers to implement yet another method. Or we can think of this as an optional feature, where only those providers looking for a serialised eviction implement it.

Reply:
I think it should fall back to the previous behavior of parallel detach if nil[empty] is returned here. The current code will skip those volumes hence skip waiting, and eventually evict pods quickly. So it's not exactly the same but very similar to the previous behavior, which seems fine to me.